ts/Series/MapLine/MapLineSeries.ts

```diff
@@ -105,12 +106,24 @@ class MapLineSeries extends MapSeries {
         point: MapLinePoint,
         state: StatesOptionsKey
     ): SVGAttributes {
+        const { mapView } = point.series.chart;
         const attr = MapSeries.prototype.pointAttribs.call(
             this,
             point,
             state
         );
 
+        if (state === 'hover') {
+            if (point.series.options.states &&
+                point.series.options.states.hover) {
+                let lineWidth = point.series.options.states.hover.lineWidth;
+
+                if (mapView && lineWidth) {
```

Comment:
`lineWidth` should work with`0`. I see no practical usage for this, but in general, the value of `0` should be allowed.